webpack/react_app/components/JobWizard/steps/Schedule/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import {
+  Title,
+  TextInput,
+  Grid,
+  GridItem,
+  Button,
+  Form,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { SelectField } from '../form/SelectField';
+import { ScheduleType } from './ScheduleType';
+import { QueryType } from './QueryType';
+import { StartEndDates } from './StartEndDates';
+
+const Schedule = () => {
+  const repeatTypes = [
+    __('Does not repeat'),
+    __('Cronline'),
+    __('Monthly'),
+    __('Weekly'),
+    __('Daily'),
+    __('Hourly'),
+  ];
+
+  const [repeatType, setSepeatType] = useState(repeatTypes[0]);
+  const [repeatAmount, setRepeatAmount] = useState('');
+  const [starts, setStarts] = useState('');
+  const [ends, setEnds] = useState('');
+  return (
+    <Form>
+      <Title headingLevel="h2">{__('Schedule')}</Title>
+      <ScheduleType />
+      <Grid>
+        <GridItem span={6}>
+          <SelectField
+            fieldId="reapet-select"
+            options={repeatTypes}
+            setValue={setSepeatType}
+            value={repeatType}
+          />
+        </GridItem>
+        <GridItem span={1} />
+        <GridItem span={5}>
+          <TextInput
+            id="repeat-amount"
+            value={repeatAmount}
+            type="number"
+            onChange={setRepeatAmount}
+            placeholder={__('Repeat N times')}
+          />
```

Comment:
Only non-negative numbers make sense here, but it seems to accept anything

Reply:
Also this should be hidden if the `Does not repeat` is selected in the schedule dropdown